app.js

```diff
@@ -132,6 +93,7 @@ function dragDrop(e) {
   e.target.remove();
   // e.target.append(draggedElement);
   changePlayer(); // how can the programme call the revertIds function already here if it's written only further below in the script?
+  // TODO(martin): To answer your question: Javascript interpreter first goes through the whole file and reads top level decalrations, which are `function ...`, without going into them. So first it learns about all the functions and remembers where they are. Then, it goes line by line and executes it all. Meaning that function F1 can call F2 even if F2 is below it, because at that point Javasccript already knows where to find it. This is called something like "2-step evluation/compilation/interpretation/something I can't remember exactly". This doesn't work for global varaibles though if I am correct, just functions I think (and classes probably).
```

Comment:
@Martinsos That's super helpful, thanks for explaining! By the way, would you get a notification about this comment even if I don't "@" you? I need to understand better how GitHub works. :D

Reply:
I am not 100% sure, but I think I would because I am the author of this PR!